boards/jiminy-mega256rfr2/Makefile.include

```diff
@@ -0,0 +1,23 @@
+include $(RIOTBOARD)/common/arduino-atmega/Makefile.include
+
+# define the cpu used by the jiminy board
+export CPU = atmega256rfr2
+
+USEMODLUE += periph
+
+# export needed for flash rule
+export PORT_LINUX ?= /dev/ttyACM0
+export PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbmodem*)))
+
+# Serial Baud rate for Ffasher is configured to 500kBaud
+# see /usr/include/asm-generic/termbits.h for availabel baudrates on your linux system
+export PROGRAMMER_SPEED ?= 0010005
+
+export FFLAGS += -p atmega256rfr2
+
+# refine serial port information for pyterm
+export BAUD = 250000
```

Comment:
For me this does not run stable (38400 does). When using shell applications and typing a command, the last character is often ignored.